qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -0,0 +1,179 @@
+import logging
+from StringIO import StringIO
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+
+class XFSTestsDev(CephFSTestCase):
+    CLIENTS_REQUIRED = 3
```

Comment:
This is why the kclient tests fail. You are asking for 3 clients but only one is available:

````
pdonnell@aglarond ~/ceph/qa/suites/kcephfs/cephfs/clusters$ cat 1-mds-1-client.yaml 
roles:
- [mon.a, mgr.y, mds.a, mds.c, osd.0, osd.1, osd.2, osd.3]
- [mon.b, mon.c, mgr.x, mds.b, osd.4, osd.5, osd.6, osd.7]
- [client.0]
````

Reply:
If three clients were absolutely necessary, you would need to add two clients to the test by appending two clients to the `roles` array. (I think that's valid...)

Reply:
Yeah, I anticipated an issue like this but wasn't sure when I mentioned about this on standup.